lib/auth/scram.js

```diff
@@ -356,9 +355,13 @@ ScramSHA.prototype.auth = function(server, connections, db, username, password,
     );
   };
 
-  var _execute = function(_connection) {
-    process.nextTick(function() {
-      executeScram(_connection);
+  var _execute = function (_connection) {
```

Comment:
1. since you are here, replace `var` with `const`
2. use an `=>` function
3. fix any linting errors